en/docs/design-your-project/index.md

```diff
@@ -90,50 +90,6 @@ To set up a Ballerina project as a VS Code workspace, add all the Ballerina pack
-
-Once you open the Architecture View, you will see the following types of diagrams.
-
-<img src="https://wso2.com/ballerina/vscode/docs/img/visual-programming/architecture-view/architecture-diagram/diagram-types.gif" class="cInlineImage-full"/>
-
-### Service Diagram Level 1
-
-This diagram displays all the services available within the workspace with links representing any interactions between them. The basic Service Diagram (Level 1) generated for the 10 Ballerina packages in the Online Boutique microservices demo project is shown below.
-
-<img src="https://wso2.com/ballerina/vscode/docs/img/visual-programming/architecture-view/architecture-diagram/service-level-one-copy.png" class="cInlineImage-full"/>
-
-!!! Info
-    The link between the `Checkout` service and the `Email` service indicates that a resource of the `Checkout` service invokes the `Email` service. In addition to this, the `Email` service also invokes an `external` service via a connector.
-
-### Service Diagram Level 2
-
-The detailed Level 2 Service Diagram (resources) dives deeper into the compositions of the services and the interactions between them. The below is the detailed representation of the demo project, which depicts the individual resource/remote functions of the services and their invocations.
-
-<img src="https://wso2.com/ballerina/vscode/docs/img/visual-programming/architecture-view/architecture-diagram/service-level-two-copy.png" class="cInlineImage-full"/>
-
-The data types of the request and response bodies of the interactions can be viewed by hovering over a particular invocation.
-
-<img src="https://wso2.com/ballerina/vscode/docs/img/visual-programming/architecture-view/architecture-diagram/level-two-invocation-data-types.gif" class="cInlineImage-full"/>
-
-If these data types are Ballerina records, click on them to access the composition of the particular record.
-
-<img src="https://wso2.com/ballerina/vscode/docs/img/visual-programming/architecture-view/architecture-diagram/level-two-to-type-composition.gif" class="cInlineImage-full"/>
-
-### Type Diagram
-
-The Type Diagram provides a comprehensive view of all the Ballerina record types defined in the project workspace and their associations. For more details, see [Type Diagram](https://wso2.com/ballerina/vscode/docs/visual-programming/architecture-view/type-diagram/).
-
-## Other capabilities of the Architecture View
-
-In addition to the above, the Architecture View includes the following features.
-
-- Filter the services and types based on the packages
-- Navigate to the source code from diagram components
-- Rearrange the diagram
-- Export the diagrams in JPEG format
\ No newline at end of file
+    <img src="https://wso2.com/ballerina/vscode/docs/img/visual-programming/architecture-view/architecture-diagram/architecture-view-cmd.gif" class="cInlineImage-full"/>
```

Comment:
I don't think we need a gif for this and devs usually don't use right click to get the command pallet.

Reply:
Removed the GIF.